server/src/com/sixsq/slipstream/ssclj/resources/spec/device.cljc

```diff
@@ -38,6 +38,9 @@
 (s/def :cimi.device/powerPlugged boolean?)
 (s/def :cimi.device/agentType ::cimi-core/nonblank-string)
 (s/def :cimi.device/actuatorInfo ::cimi-core/nonblank-string)
+(s/def :cimi.device/sensorType ::cimi-core/nonblank-string)
```

Comment:
this is not resolved. if this information is not supposed to be in device.cljc, then remove it please

Reply:
resolved